tests/api_tests/test_auth.py

```diff
@@ -44,28 +45,37 @@ def test_login_and_logout(auth_headers):
     ), "The IDs of the new user and the registered user do not match."
 
     logger.info("Logout with new user")
-    rs_logout = request_utility.post(
-        endpoint="users/logout",
+    url = RequestUtilities.get_base_url() + "users/logout"
+    response_logout = requests.post(
+        url=url,
+        json={"email": user_info["email"], "password": user_info["password"]},
         headers={"Authorization": f"Bearer {user_rs_api['token']}"},
+        timeout=10,
     )
 
-    assert not rs_logout, "Request is not empty."
+    assert (
+        response_logout.status_code == 200
+    ), "Bad status code of logout response."
```

Comment:
In the 21st century no need to make a string so short, if linters fails - adjust them.
Acceptable lengh for a line is 100-120 symbols

```suggestion
    assert(response_logout.status_code == 200), "Bad status code of logout response."
```